websites/docs/docusaurus.config.js

```diff
@@ -57,6 +57,15 @@ const config = {
               : defaultUrl
             return editUrl
           },
+          remarkPlugins: [
+            [
+              require('@docusaurus/remark-plugin-npm2yarn'),
+              {
+                sync: true,
```

Comment:
Users can synchronize tab containing package manager name used by themselves in our all pages of official docs by npm2yarn's `sync` option.

https://docusaurus.io/docs/next/markdown-features/code-blocks#npm2yarn-remark-plugin-configuration